VERSION

```diff
@@ -1 +1 @@
-1.7.5
+1.7.6
```

Comment:
I think this is a big enough change to warrant a minor version bump, so we can make it version 1.8.0.

Reply:
Bumped